### DISCUSSION TITLE
running errors with Start.R scripts

### DISCUSSION THREAD

**Discussion by dengzhong** [magpiemodel/magpie#433] [Help] [answered]
Dear Jan Dietrich,
When I was running the Start.R script, I got the error below. Could you please guide me on how to solve this issue? Thanks!
Starting MAgPIE...
[1] 127
Warning message:
In system(paste("gams full.gms -errmsg=1 -lf=full.log -lo=", cfg$logoption,  
'gams' not found

I found this line of command (system(paste("gams full.gms -errmsg=1 -lf=full.log -lo=",cfg$logoption,sep=""))) appears in the script of submit.R, My understanding is that R did not find the GAMS software, if my understanding is correct, is there any modification needed in the submit.R script?

**Comment by tscheypidi** [comment, marked as answer]
You need to make sure that the `gams` executable can be found system wide. To do so you need to add the path to your GAMS installation into the `PATH` variable of your system. How to add a path to the `PATH` variable depends on the system you are using, but there are vairous tutorials online explaining how to do it in case you have not done it before.
